elements/rh-card/src/rh-card.scss

```diff
@@ -2,10 +2,30 @@
 
 :host {
   --rh-card--padding:                          calc(#{rh-var(container-spacer)} * 2); // 2x `container-spacer`
-  --rh-card_header--size:                      #{rh-var(font-size--heading--gamma)};
+  --rh-card--paddingTop:                       var(--rh-card--padding);
+  --rh-card--paddingRight:                     var(--rh-card--padding);
+  --rh-card--paddingBottom:                    var(--rh-card--padding);
+  --rh-card--paddingLeft:                      var(--rh-card--padding);
+  // Spacing between elements inside the card
+  --rh-card--spacing:                          #{rh-var(container-spacer)};
+
+  // bg is deprecated but still supported
   --rh-card--bg:                               #{rh-color(surface--base)};
+
+  // Color properties
+  --rh-card--backgroundColor:                  var(--rh-card--bg);
+  --rh-card--borderColor:                      transparent;
+
+  // Header properties
```

Comment:
I want to play with this more but leaving a note here so I don't forget, I think these variables should be `broadcasted` instead of explicit